generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,408 @@
+
+</div>
+
+Linear models are so popular in statistics because equations for lines are computationally simple (even if they're not simple to understand!) but still allow us to model pretty complex relationships.
+
+It would be handy if linear models covered all possible research questions, but unfortunately they don't!
+**Generalized linear models are a way to extend the functionality of linear models to cover situations when they normally wouldn't work.**
+
+## Why do we need generalized linear models?
+
+Linear models are only appropriate when your outcome variable(s) are **continuous** and **unbounded**. 
+If you want to model a categorical outcome, or one that's bounded (like a proportion or percent), then usually can't rely on regular linear modeling to get the job done. 
+
+<h3>What counts and continuous and unbounded?</h3>
+
+A variable is **continuous** if it can take any value on a scale, not just discrete options. 
+For example, age is generally continuous because you can be 11 years old, or 11.5, or 11.476 years old, etc.
+In contrast, experimental group (treatment vs. control) is not continuous; there's no way to be halfway between treatment and control, you're in one or the other. 
+Something like medication dose could be continuous or categorical depending on how it's used. 
+If patients are restricted to categorical levels of dose (e.g. high, med, or low dose groups) then it would be categorical. 
+But if it's something where their dose might theoretically be anywhere on the scale, then it's continuous.
+
+A variable is **unbounded** if it can theoretically extend from negative infinity at the low end to positive infinity at the high end. 
+We often have variables that aren't truly unbounded, but they're close enough for all practical purposes -- for example, IQ can't technically be negative, but in practice no measured IQs are close to 0 so the data stop naturally before they hit a the scale boundary at 0. 
+IQ is generally treated as unbounded statistically.
+
+Something like a probability or proportion is bounded at 0 and 1; it's not possible to have a negative probability, or one over 1. 
+
+A variable like age might be unbounded or bounded depending on what ages you're studying. 
+In a young pediatric population, age is bounded at 0 because you can't have any negative ages and a distribution of infant ages will get cutoff at that lower bound. 
+But if you're studying adults, then age might be functionally unbounded in your dataset.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For an excellent review of continuous vs. categorical variables, see the [Khan Academy video on discrete and continuous random variables](https://www.khanacademy.org/math/statistics-probability/random-variables-stats-library/random-variables-discrete/v/discrete-and-continuous-random-variables).
```

Comment:
This link seems to be broken.

Reply:
Do you mind checking this again? This link is working for me...

Reply:
Hmm, it is working for me too now, don't know what was going on before!

Reply:
Cool! I love problems that fix themselves :)